games/a.yaml

```diff
@@ -1328,6 +1328,26 @@
   video:
     youtube: ysY9vKKisbo
 
+- name: Atari Asteroids
+  originals:
+  - Asteroids
+  type: clone
+  repo: https://github.com/TheBeachLab/asteroids
+  url: https://web.archive.org/web/20140104211104/http://www.edge-online.com/features/making-asteroids/
```

Comment:
```suggestion
```
I think this is unrelated to the game itself, it is only recreated in the project's readme